h/services/rename_user.py

```diff
@@ -57,6 +58,11 @@ def _purge_auth_tickets(self, user):
             .filter(models.AuthTicket.user_id == user.id) \
             .delete()
 
+    def _purge_tokens(self, user):
+        self.session.query(models.Token) \
+            .filter(models.Token.userid == user.userid) \
```

Comment:
Hmm. Perhaps we could just keep the tokens valid but update the userid everywhere instead? The problem with doing this for auth tickets, as I discovered yesterday, is that the userid is present in the value stored in the user's browser. I haven't looked into whether that is fixable.

If the only reference to the userid for OAuth tokens is in our DB then we could just update it rather than delete them.

Thoughts?